### DISCUSSION TITLE
博士论文模版中英文研究方向设置无效

### DISCUSSION THREAD

**Discussion by Nineteen-Hundred** [note286/xduts#17] [Q&A] [answered]
英文研究方向（如xxxx in Engineering）设置后Engineering并不显示，在文档里能看到该处只有专硕才会显示，希望能进行修改。

**Comment by note286** [comment]
请具体描述，你是学术博士还是专业博士？最好图示，你可以在Microsoft Word模板中找到对应位置截图。

**Comment by Nineteen-Hundred** [comment]
我是学术博士，这个地方的in后面应该是有一个具体的研究领域。第一张图是模版，后面两张是我找的别人的论文。也见过别人不放的，不过都会空着，而不会放一个in，这个地方应该是一个bug，还希望能够修改一下，感谢了！
<img width="425" alt="QQ20221201-122556@2x" src="https://user-images.githubusercontent.com/32399176/204966085-fef9de2d-af92-4a2e-87e6-8f6276205448.png">

<img width="558" alt="QQ20221201-122725@2x" src="https://user-images.githubusercontent.com/32399176/204966125-59e5add8-dabc-448d-bd87-b003ca7adb98.png">

<img width="520" alt="QQ20221201-122742@2x" src="https://user-images.githubusercontent.com/32399176/204966147-fd773456-0d19-4cea-a9c7-8180b1d78b3f.png">

**Reply by JiaxiangHuang** [reply]
major*={Engineering} 这个地方写你的二级学科，也就是你的研究方向，看了几个范文工科都这么写。其实note286把这个major*={Engineering} 改成sub-major*={Engineering} 会更好一点

**Reply by note286** [reply]
@JiaxiangHuang 此处完全是根据学位办的Microsoft Word模板设置的，是一级学科的英文名称，如有疑问，请给出相应的依据。另外，在西安电子科技大学的研究生学位论文中，是不会出现二级学科的英文名称的。

**Comment by note286** [comment, marked as answer]
```latex
\documentclass{xdupgthesis}
\xdusetup{info/graduate-type={博士},info/degree-type={学术},info/major*={Education Science}}
\begin{document}
\frontmatter
\mainmatter
\backmatter
\end{document}
```
![image](https://user-images.githubusercontent.com/16207292/204968462-ba61809b-270a-4fd8-9ed7-4060edb975eb.png)
我没有发现什么异常。
